scripts/release_scripts/repo_specific_changes_fetcher_test.py

```diff
@@ -103,22 +99,26 @@ def mock_run_cmd(unused_cmd: str) -> str:
         self.assertEqual(actual_storgae_models, expected_storage_models)
 
     def test_get_changes(self) -> None:
+        call_counter = 0
         def mock_get_changed_schema_version_constant_names(
             unused_release_tag_to_diff_against: str
         ) -> Optional[List[str]]:
-            if self.call_counter:
+            nonlocal call_counter
```

Comment:
Same question, why not use the existing constructs already defined in test_utils rather than introducing a new pattern?

Reply:
Ditto as above.